mne/viz/evoked.py

```diff
@@ -2241,7 +2241,8 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
         ch_names = evokeds[cond][0].ch_names
         picks = range(len(ch_names))
 
-    info = pick_info(info, picks, True)
+    info = pick_info(info, pos_picks, True)
+    all_ch_names = ch_names if ch_type == 'grad' else info['ch_names']
```

Comment:
the `picks` vs `pos_picks` was the cause of RMS-combined grads being in the wrong locations